src/components/MyKiva/LendingStats.vue

```diff
@@ -1,14 +1,105 @@
 <template>
-	<div>
-		<h3
-			class="tw-text-primary"
-		>
+	<div class="tw-mb-3.5">
+		<h3 class="tw-text-primary tw-mb-1">
 			Ready to grow your impact?
 		</h3>
-		<p
-			class="tw-text-base"
-		>
+		<p class="tw-text-base">
 			Next steps for you based on your lending history
 		</p>
 	</div>
+	<div class="tw-bg-white tw-rounded tw-shadow tw-p-1 md:tw-p-2 tw-w-full" style="max-width: 800px;">
```

Comment:
I think instead of using custom width here, we can use tailwind basis classes
https://tailwindcss.com/docs/flex-basis

Reply:
Yeah we can see how we tweak this width when the other element is added next to it.